src/main/java/com/exasol/RequestDispatcher.java

```diff
@@ -0,0 +1,91 @@
+package com.exasol;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import org.apache.commons.cli.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.exasol.github.*;
+import com.exasol.release.ReleasePlatform;
+
+/**
+ * This class is the main entry point for calls to a Release Robot.
+ */
+public class RequestDispatcher {
```

Comment:
Since this is the entry point, I suggest, you call it `ReleaseRobot`. :-)

Reply:
Nice suggestion!